db/seeds.rb

```diff
@@ -1,14 +1,23 @@
+user = User.create(
+  email: "capstonerman@gmail.com",
+  password: "123Rman!",
+  username: "capstonerman")
+user2 = User.create(
+  email: "notarthur@gmail.com",
+    password: "123Arthur!",
+    username: "arthur-look-a-like")
+
+
+
 posts = [
   {
-    id: 1,
     user_id: 1,
```

Comment:
Sorry I didn't see this earlier. Now that the association will be created through the create method ` user.posts.create(post)` please remove all the user_id from the posts. The foreign key is automatically assigned after the method is ran.